client/.gitignore

```diff
@@ -1,2 +0,0 @@
-node_modules
```

Comment:
I moved the .gitignore to the root and added node_modules there.

The previous team used mailers to let students know they've been added to a course. We'll probably also need mailers for forgot password. I can take a look through it again but the only thing I can think of removing is action cable if we aren't going to be using web sockets for anything.